app/store/localStorage.js

```diff
@@ -57,6 +57,45 @@ export function removeItem(k) {
   }
 }
 
+export function getCustomizedSettings() {
+  return getItemAsJson('customizedSettings');
+}
+
+export function setCustomizedSettings(data) {
+  // Get old settings and test if set values have changed
+  const oldSettings = getCustomizedSettings();
+  const newSettings = {
+    accessibilityOption: data.accessibilityOption ? data.accessibilityOption
+    : oldSettings.accessibilityOption,
+    minTransferTime: data.minTransferTime ? data.minTransferTime
+    : oldSettings.minTransferTime,
+    modes: data.modes ? data.modes
+    : oldSettings.modes,
+    walkBoardCost: data.walkBoardCost ? data.walkBoardCost
+    : oldSettings.walkBoardCost,
+    walkReluctance: data.walkReluctance ? data.walkReluctance
+    : oldSettings.walkReluctance,
+    walkSpeed: data.walkSpeed ? data.walkSpeed
+    : oldSettings.walkSpeed,
+  };
```

Comment:
Maybe this verbose data merging block could be expressed more compactly using object spread or another (lodash etc) merger?